### DISCUSSION TITLE
刷委托时是否应该把道中全部打完？

### DISCUSSION THREAD

**Discussion by ec7529a2** [LmeSzinc/AzurLaneAutoScript#3712] [Ideas]
闲得没事突然想到，根据alas的刷委托任务设置界面里给出的[紧急委托刷新机制](https://bbs.nga.cn/read.php?tid=27134956)，平均每场道中获得0.03个委托，打boss获得0.075个委托，进主线图获得0.005个委托，进活动图获得0.0025个委托。
简单计算可得，进图平均每油获得0.005 ÷ 10 = 0.0005个委托，血亏。打boss平均每油获得0.075 ÷ 3 = 0.025个委托，血赚。（注意现在退役不退油了）
所以，**只要进图就要打boss**，那么我们就可以把进图和打boss视为同一个操作的两个步骤。也就是说，进图并打boss，平均每油获得(0.005 + 0.075) ÷ (10 + 3) ≈ 0.006个委托。
但是道中平均每油可以获得0.03 ÷ 3 = 0.01个委托，**比“进图并打boss”这个操作更赚**。所以理论上刷委托时不应该做“进图并打boss”这个操作，而应该只打道中。
但是实际上，不进图就不能打道中，所以实际上刷委托时应该尽可能提高打道中的次数，降低“进图并打boss”的次数，也就是**把道中全部打完**，又称全清，然后再去打boss。

以2-4为例，如果每次进图只打3场道中（假设没被堵），则平均每油获得(0.005 + 0.03 × 3 + 0.075) ÷ (10 + 3 × 3 + 3) ≈ 0.0077个委托。
但如果每次进图打7场道中（即全清），则平均每油获得(0.005 + 0.03 × 7 + 0.075) ÷ (10 + 3 × 7 + 3) ≈ 0.0085个委托，比只打3场道中赚约10％
再以这次活动（μ兵装三期）的SP2为例，如果每次进图只打4场道中，则平均每油获得(0.0025 + 0.03 × 4 + 0.075) ÷ (10 + 3 × 4 + 3) = 0.0079个委托。
但如果每次进图打8场道中，则平均每油获得(0.0025 + 0.03 × 8 + 0.075) ÷ (10 + 3 × 8 + 3) ≈ 0.0086个委托，比只打4场道中赚约8.6％
再以上一个大活动（深度回音复刻）的A3和B1为例，A3每次进图分别打4场和7场道中，平均每油分别获得约0.0079个和0.008456个委托，全清赚约7％
B1每次进图分别打4场和10场道中，（假设委托概率与A3相同）平均每油分别获得约0.0079个和0.00878个委托，全清赚约11.1％，B1全清比A3全清赚约3.8％

不用担心空弹打不过，因为跨队支援现状如何，在座的各位有目共睹。而且alas还会换装备，装备码功能也在做了，更是进一步降低了翻船率。
如果以上内容有错误，请务必指出并详细说明，谢谢。

**Comment by Air111** [comment]
时间效率会低不少

空弹能全清的图其实很有限，1油前排也要负责清理掉前置，而且空弹遇到大航也有不小的翻车率

**Comment by ec7529a2** [comment]
我又想到一个更离谱的刷法：刷伏击
假设平均每场伏击获得的委托数量与道中相同，就可以关闭周回模式用这样的编队来刷伏击：
常规3油低耗队带莫里猫，旗舰带维修加对抗模拟，另一队带1级无装备但升满广域舰船维护的英仙座，外加两艘跨队支援不限场次的战列，前排不要带高机动船。
这是为了拉满回血并压低制空和侦察值，莫里猫可以规避空袭。
但是我实际上不确定刷伏击能出多少委托，因为我看过的资料都没有单独说明**伏击**能出多少委托，稍微搜索了一下也没找到。
所以如果伏击不出委托就当我没说。
